main.py

```diff
@@ -15,9 +15,10 @@
 
 
 # ----------------- POSTGRES -----------------
+table_name = os.getenv("DB_TABLE")
```

Comment:
Constant variables should be in constant casing. 
```suggestion
TABLE_NAME = os.getenv("DB_TABLE")
```

Reply:
Resolved [here](https://github.com/acmcsufoss/acmOpportunities/pull/19/commits/0a25691c94fa80f551275e70dec5c612b0014058)